src/models/rpcModel.ts

```diff
@@ -0,0 +1,147 @@
+export enum RpcMethod {
+  GET_BTC_BALANCE = 'get_bitcoin_balance',
+  WITHDRAW_BTC = 'withdraw_btc',
+  BUY_XMR = 'buy_xmr',
+  RESUME_SWAP = 'resume_swap',
+  LIST_SELLERS = 'list_sellers',
+  CANCEL_REFUND_SWAP = 'cancel_refund_swap',
+  GET_SWAP_INFO = 'get_swap_info',
+  SUSPEND_CURRENT_SWAP = 'suspend_current_swap',
+  GET_HISTORY = 'get_history',
```

Comment:
**Suggestion:** Consider using a more descriptive name for the enum `RpcMethod`. The current name is not very descriptive of what the enum represents. A more descriptive name could be `RpcSwapMethods` or `RpcSwapActions`.
```suggestion
export enum RpcSwapMethods {
  GET_BTC_BALANCE = 'get_bitcoin_balance',
  WITHDRAW_BTC = 'withdraw_btc',
  BUY_XMR = 'buy_xmr',
  RESUME_SWAP = 'resume_swap',
  LIST_SELLERS = 'list_sellers',
  CANCEL_REFUND_SWAP = 'cancel_refund_swap',
  GET_SWAP_INFO = 'get_swap_info',
  SUSPEND_CURRENT_SWAP = 'suspend_current_swap',
  GET_HISTORY = 'get_history',
}
```